Dockerfile

```diff
@@ -53,6 +53,11 @@ COPY --from=packages /packages/install/ /packages/install/
 # Copy select binary packages
 COPY --from=packages /dist/ /usr/local/bin/
 
+# Copy helmfiles and kiam scripts
```

Comment:
We probably don't need to copy `helmfiles` into `geodesic` image.
We do it in reference architecture modules like this https://github.com/cloudposse/testing.cloudposse.co/blob/master/Dockerfile#L3
@osterman what do you think?